.github/workflows/tests.yml

```diff
@@ -25,7 +25,7 @@ jobs:
               run: "yarn install --frozen-lockfile"
 
             - name: Run tests
-              run: yarn test --coverage
```

Comment:
This should also generate the coverage for the Synapse tests, otherwise the Sonar job which happens after `Tests` won't be able to pick them up